lib/logstash/filters/mutate.rb

```diff
@@ -233,6 +245,18 @@ def remove(event)
   end # def remove
 
   private
+  def select(event)
+    replacement = LogStash::Event.new
+
+    @select.each do |field|
+      field = event.sprintf(field)
+      next unless event.include?(field)
+      replacement[field] = event[field]
+    end
+    event.overwrite replacement
```

Comment:
@purbon I think you're right in that it will overwrite `@timestamp` and `@version`, which isn't ideal. Any suggestions on how to prevent that from happening?

I skipped all fields starting with `@` in a previous iteration, but @talevy pointed out that `@timestamp` and `@version` are the only reserved fields.